tools/wltests/android/create_boot_img.sh

```diff
@@ -64,6 +64,12 @@ if [ "${ANDROID_RAMDISK_OFFSET}" ]; then
     ANDROID_RAMDISK_OFFSET="--ramdisk_offset ${ANDROID_RAMDISK_OFFSET}"
 fi
 
+if [ "${NO_RAMDISK}" ]; then
```

Comment:
The point is that variables are empty by default in Bash... thus, you probably don't need to be "explicit"...

However, looking better into it I was wondering why we cannot use an empty RAMDISK definition to just say "i don't want a ramdisk" ?

If you just set in your definitions file:
```bash
RAMDISK_IMAGE=""
```
this will be turned into a:
```bash
android/create_boot_img.sh:DEFAULT_RAMDISK="${PLATFORM_OVERLAY_PATH}/${RAMDISK_IMAGE}"
```
and finally, in your:
```bash
RAMDISK="${RAMDISK:-$DEFAULT_RAMDISK}"
```

Which, ultimately means that all this patch can be simply reduced to this:

```bash
[ -f ${RAMDISK} ] || ramdisk_cmd="--ramdisk '${RAMDISK}'"
```

Meaning that if we do not provide somehow a ramdisk... we don't want to use it... without any additional explicit NO_RAMDISK variables ;)

Reply:
The reason I added that explicit `NO_RAMDISK` was to differentiate
- when there is no provided ramdisk but we **need** one
- when there is no provided ramdisk and we **know** it's okay

So while setting `RAMDISK_IMAGE=""` can do that, not specifying `RAMDISK_IMAGE` at all will have the same behaviour - in which case we won't warn the user that he/she forgot to specify it.

Reply:
I don't see this issue for the definitions file: if you forget to specify a ramdisk in that file for a platform which actually needs it and you are sending a pull request... you deserve to be punished by not getting a warning.

IOW: why you should be wise enough to remember to add `NO_RAMDISK` but, at the same time, stupid enough to not specify a ramdisk if you really need it?

I would really keep it simple as assume that who contributes a platform support does not requires warning for things he supposed to specify... unless I'm missing a use-case, you can better explain be tomorrow over a coffee ;-)